apps/question-bank/src/main/resources/application.yml

```diff
@@ -6,7 +6,7 @@ nbs:
   datasource:
     server: localhost
   security:
-    tokenSecret: qJ5+oDurrJnxfaAz6h4Eg1saP7NSeHuQQ/Yq4vh+mbi4B4QYThc/M+33VtPJ5O2A
+    tokenSecret: ${TOKEN_SECRET}
```

Comment:
Will this require new secrets for helm charts? Assuming yes, if they do not already exist.

Reply:
It appears we are not setting the `tokenSecret` or `parameterSecret` via the helm chart. We will need to do that.